### PR TITLE
Db Nodes action cleanup

### PR DESCRIPTION
**Pull request by jhorvath**
There were inappropriate actions at DB nodes in VSCode. We need connect/disconnect just on connection nodes, not on table nodes, etc. Also adding RefreshAction to VScode. For this I changed action registration from layer to `@ActionRegistration` annotation.

### CODE REVIEW

File: ide/db/src/org/netbeans/modules/db/explorer/action/RefreshAction.java [hunk 1]
```diff
@@ -35,6 +40,25 @@
...
  * @author Rob
  */
+@ActionRegistration(
+        displayName = "#Refresh", 
+        lazy = false,
+        enabledOn = @ActionState(type = DatabaseConnection.class, useActionInstance = true)
+)
+@ActionID(category = "Database", id = "netbeans.db.explorer.action.Refresh")
+@ActionReferences(value = {
+    @ActionReference(path = "Databases/Explorer/Connection/Actions", position = 100),
```
Comment (sdedic) [inline ide/db/src/org/netbeans/modules/db/explorer/action/RefreshAction.java:50]: The action had originally different positions in the `Databases/Explorer/**/Actions" - is the reorder OK now ? It seems that it fit in between some separator-divided groups with the original position = 350 (?)

File: java/java.lsp.server/vscode/package-lock.json [hunk 2]
```diff
@@ -1,1232 +1,8 @@
 {
 	"name": "apache-netbeans-java",
 	"version": "0.1.0",
-	"lockfileVersion": 2,
```
Comment (sdedic) [inline java/java.lsp.server/vscode/package-lock.json:4]: This seems like a newer version of NPM was used - @dbalek OK ?

File: java/java.lsp.server/vscode/package.json [hunk 3]
```diff
@@ -648,10 +656,14 @@
...
 					"command": "nbls:Database:netbeans.db.explorer.action.Connect",
-					"when": "viewItem =~ /class:ddl.DBConnection/"
+					"when": "viewItem =~ /class:org.netbeans.api.db.explorer.DatabaseConnection/"
 				},
 				{
 					"command": "nbls:Database:netbeans.db.explorer.action.Disconnect",
+					"when": "viewItem =~ /class:org.netbeans.api.db.explorer.DatabaseConnection/"
+				},
+				{
+					"command": "nbls:Database:netbeans.db.explorer.action.Refresh",
```
Comment (sdedic) [inline java/java.lsp.server/vscode/package.json:666]: Q: should the `ddl.DBConnection` remain here, or should it be replaced with `DatabaseConnection` as well ?

File: java/java.lsp.server/vscode/package.json [hunk 4]
```diff
@@ -648,10 +656,14 @@
 				},
 				{
 					"command": "nbls:Database:netbeans.db.explorer.action.Connect",
-					"when": "viewItem =~ /class:ddl.DBConnection/"
+					"when": "viewItem =~ /class:org.netbeans.api.db.explorer.DatabaseConnection/"
```
Comment (sdedic) [inline java/java.lsp.server/vscode/package.json:659]: Great, `DatabaseConnection` is a public API of DB explorer...

File: java/java.lsp.server/vscode/package.json (hunk 3 above)
Comment (jhorvath) [reply java/java.lsp.server/vscode/package.json:666]: It should remain here as ddl.DBConnections is on all other nodes where Refresh action is needed

File: java/java.lsp.server/vscode/package-lock.json (hunk 2 above)
Comment (jhorvath) [reply java/java.lsp.server/vscode/package-lock.json]: Fixing in 40a21f6

File: ide/db/src/org/netbeans/modules/db/explorer/action/RefreshAction.java (hunk 1 above)
Comment (jhorvath) [reply ide/db/src/org/netbeans/modules/db/explorer/action/RefreshAction.java]: Fixing in 40a21f6

**Review by sdedic — APPROVED** [review]

**Pull request merged**